### ISSUE TITLE
Quick fix

### ISSUE THREAD

**Issue by kocmitom** [thompsonb/prism#3]
Hi,
when using your code as a metric, there is a small problem:
ModuleNotFoundError: No module named 'modified_sequence_scorer'

You can fix it by adding a dot before "modified_sequence..." on line https://github.com/thompsonb/prism/blob/master/prism.py#L18

> from .modified_sequence_scorer import SequenceScorer

Thank you,
Tom

**Comment by thompsonb** [comment]
What did you do, exactly? Trying to replicate.

**Comment by kocmitom** [comment]
I am trying to use prism inside my framework, therefor I have cloned it to my folder, and imported into my python files. But then if I try to run it, it will fail because it cannot find "modified_sequence_scorer" in my root folder from where I am running it.
This is a list of my root folder:

my_code/
prism/
main.py

So the modified_sequence_scorer file is inside the prism folder, which is a problem when I am running it from a parent folder. Maybe there are other solutions except for the dot, I just found this one worked.
